msticpy/sectools/domain_utils.py

```diff
@@ -226,7 +226,7 @@ def in_abuse_list(self, url_domain: str) -> Tuple:
             )
         except Exception:  # pylint: disable=broad-except
             result = False
-            x509 = None
+            x509 = None  # type: ignore
```

Comment:
I think I fixed this in the data_viewer branch as well, maybe a bit more elaborately (ISTR importing the Certificate type)

Reply:
Seen that, Ill merge with your data_viewer fix